test/plugin/test_out_forward.rb

```diff
@@ -920,6 +1022,39 @@ def create_target_input_driver(response_stub: nil, disconnect: false, conf: TARG
     }.configure(conf)
   end
 
+  def create_certificate(name, serial, parent = nil, ca = true)
+    key = OpenSSL::PKey::RSA.new 2048
+    cert = OpenSSL::X509::Certificate.new
+
+    parent_cert = cert
+    parent_key = key
+    if parent
+      parent_cert = parent[0]
+      parent_key = parent[1]
+    end
+
+    cert.version = 2
+    cert.serial = serial
+    cert.subject = OpenSSL::X509::Name.parse "/CN="+name
+    cert.issuer = parent_cert.subject
+    cert.public_key = key.public_key
+    cert.not_before = Time.now
+    cert.not_after = cert.not_before + 60*60 # 1 hour
+    ef = OpenSSL::X509::ExtensionFactory.new
+    ef.subject_certificate = cert
+    ef.issuer_certificate = parent_cert
+    cert.add_extension(ef.create_extension("subjectKeyIdentifier","hash",false))
+    if ca
+      cert.add_extension(ef.create_extension("basicConstraints","CA:TRUE",true))
```

Comment:
space after ,. (same as L1048-L1052)
```suggestion
      cert.add_extension(ef.create_extension("basicConstraints", "CA:TRUE", true))
```